debian/changelog

```diff
@@ -0,0 +1,5 @@
+python-ev3dev (0.2.2.post1-1) unstable; urgency=low
```

Comment:
In order to upload to the ev3dev.org repository, the distribution must be `stable`. Also, since the Debian packaging is being included in the upstream repository, we don't need a debian version (that's the `-1`) at the end (see comment on `debian/source/format` below).

Also, can you explain the `.post1` part of the version number. It does not look like any version scheme I am familiar with and we need to make sure that debian package version comparisons will work correctly.